features/host-images-in-forums-with-assets.js

```diff
@@ -0,0 +1,80 @@
+var button5 = document.querySelector(".markItUpButton5");
+var bb5 = document.createElement('li');
+var img = document.createElement('img');
+img.src = 'https://github.com/lopste/forumoji/blob/main/resources/forumoji/sunrise-over-mountains.png?raw=true';
+img.alt = 'upload image icon';
+button5.before(bb5);
+bb5.classList.add("markItUpButton");
+bb5.appendChild(img);
+bb5.style.marginRight = "5px";
```

Comment:
Could you please make it so that the cursor = pointer? That way it acts like the rest of the buttons.